lib/mongoid/clients/factory.rb

```diff
@@ -87,6 +87,12 @@ def driver_version
         Mongo::VERSION.split('.')[0...2].map(&:to_i)
       end
 
+      # Prepare options for Mongo::Client based on Mongoid client configuration.
+      #
+      # @param [ Hash ] opts Parameters from options section of Mongoid client configuration.
+      # @return [ Hash ] Options that should be passed to Mongo::Client driver.
```

Comment:
Client is not really a "driver", can you s/driver/constructor/ please?

Reply:
Sorry 😊 
Fixed.